soccer/gameplay/plays/offense/adaptive_formation.py

```diff
@@ -17,6 +17,11 @@
 
 class AdaptiveFormation(standard_play.StandardPlay):
 
+
+
+    situationList = ['attack_goal', 'defend_clear', 'offensive_scramble']
```

Comment:
I personally would prefer these to be enums so I don't have to worry about misspelling them

Reply:
I think that's a fair point

Reply:
I agree

Reply:
Thirded.

Reply:
So currently both

`situationList = [ 'clear', 'defend_goal', 'defensive_scramble', 'offensive_pileup','defensive_pileup']`

and
```
situationList = [situational_play_selection.SituationalPlaySelector.Situation.CLEAR,
            situational_play_selection.SituationalPlaySelector.Situation.DEFEND_GOAL,
            situational_play_selection.SituationalPlaySelector.Situation.DEFENSIVE_SCRAMBLE,
            situational_play_selection.SituationalPlaySelector.Situation.OFFENSIVE_PILEUP,
            situational_play_selection.SituationalPlaySelector.Situation.DEFENSIVE_PILEUP ]
```

are tested and working. I can remove string support if we want; I mostly added it because I thought the enums would be long although I suppose that's fine. It does have an option to spell check the strings as a default parameter. Having to write everything out brings two things to my mind though: one should I make the enum its own file?, and also I am open to renaming some things. situational_play_selection and SituationalPlaySelector might be both too wordy and no longer accurate to what the file and class do. Situation Analysis has been what we have been calling it but I'm open to suggestions.